src/Tree/Entity/Repository/NestedTreeRepository.php

```diff
@@ -800,10 +849,20 @@ public function reorderAll($sortByField = null, $direction = 'ASC', $verify = tr
      * If any error is detected it will return an array
      * with a list of errors found on tree
      *
+     * @param array $options
+     *
+     * $options['treeRootNode']  = (object) Optional tree root node to verify, if not the whole forest (default: null)
+     *                             Option is only available for forests, not for single trees
+     *
      * @return array|bool true on success,error list on failure
      */
-    public function verify()
+    public function verify(array $options = [])
```

Comment:
I'm not using the Tree behaviour, but is it likely to pass more options? if it's just one we could maybe use `$treeRootNode`

Reply:
I agree that in this case a single `$treeRootNode` argument would be better, however I just wanted to keep consistency with other places where I added extra options. For example in the `recover()` below there are a lot optional options and one of them is again `treeRootNode`. 
In the latter case, I'd argue that it's better to have an `$options` array, at least for PHP7 (there are also other precedents in the bundle where such an `$options` array is used), but that is probably no longer the case with the named arguments in PHP8, so maybe I should use separate arguments everywhere? 
On the other hand, in the case of `getPath -> getPathQuery -> getPathQueryBuilder`, it's maybe easier to maintain with an array of options, as if you add another a new option, you don't have to change the signature of all 3 methods. WDYT?

Reply:
Hey @pmishev sorry for the late reply, yeah, I guess it's fine to keep these `$options`, please specify all these options in PHPDoc as `array{option1: type, ...}`